Test/dafny4/Leq.dfy

```diff
@@ -1,3 +1,4 @@
+// UNSUPPORTED: windows, macosx
```

Comment:
Not blocking on it if the CI passes so we can merge this, but I logged https://github.com/dafny-lang/dafny/issues/4178 which we can link to here